Source/Icebreaker/Content/styles.css

```diff
@@ -209,30 +208,27 @@ body {
     }
 
 .theme-default .primary-button-back {
```

Comment:
There's only 1 primary button. Instead of primary-button-back call the style secondary-button

Reply:
Made the necessary changes